scout/server/blueprints/variants/controllers.py

```diff
@@ -27,6 +27,7 @@
     SPIDEX_HUMAN,
     VARIANT_FILTERS,
 )
+from scout.constants.variant_tags import VARIANTS_TARGET_FROM_CATEGORY
```

Comment:
```suggestion
from scout.constants.variant_tags import VARIANTS_TARGET_FROM_CATEGORY
from scout.constants import CANCER_EXPORT_HEADER, EXPORT_HEADER
```

Reply:
Ahem, seems we already forgot that before, but we did do the import to `scout.constants` with `CANCER_EXPORT_HEADER` etc so we might as well.